### PR TITLE
Create aac-contribute

### PR DESCRIPTION
**Pull request by EdPrice-MSFT**
# Contributor guide pull request

Thanks for contributing to the [docs.microsoft.com contributor guide](https://docs.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.

## Quality control

- [ ] 1. **Successful build with no warnings**: Review the build status to make sure **all checks are green** (Succeeded).

- [ ] 2. **#Sign-off**: Once the PR is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the PR. Signing off means the document is ready for review and can be published at any time.

## Merge and publish

- All PRs to this repository are manually reviewed and merged.
- Once all feedback on the PR is addressed, we'll merge the PR into the main branch.

## Need help?

- See our guidance on [Contributing to the contributor guide](/contribute/?branch=main) for help with adding or updating content in this contributor guide.
- Write to the PR reviewer in the PR comments: `@carlyrevier, @jehchow`

### CODE REVIEW

**Comment by EdPrice-MSFT** [comment]
In progress

**Comment by EdPrice-MSFT** [comment]
#Sign-Off 

This adds the new Documentation set-specific guidance for AAC!

File: Contribute/architecture-center/aac-contribute.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+author: EdPrice-MSFT
+ms.author: edprice
+ms.date: 07/08/2022
+---
+
+# Azure Architecture Center contributions
+
+You can edit articles or create new articles for the Azure Architecture Center. The Azure Architecture Center (AAC) encourages contributions from the public, through GitHub issues and pull requests (PR). This page covers the general guidelines for contributing content to the Azure Architecture Center.
+
+## What is the Azure Architecture Center
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:16]:
```suggestion
## What is the Azure Architecture Center?
```
I think this is ok as a question, but consider making this H2 simply "Overview".

File: Contribute/architecture-center/aac-contribute.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+If your solution includes code and/or a deployment, then there are a number of additional requirements.
+
+### Process
+
+1. You (the contributor) must submit a proposed article. Send an email to [AACProposals@microsoft.com](mailto:AACProposals@microsoft.com). Include the following information:
+    1. Your name.
+    1. The name of your proposed article.
+    1. What type of content this should be. Options include an Architecture Guide, a Solution Idea, an Example Workload, or a Reference Architecture.
+    1. A brief description of your article.
+    1. The category (such as AI/ML, Databases, or Security) and/or the industry (such as Retail, Healthcare, and so on) that this article would fit in (
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:53]:
```suggestion
    1. The category (such as AI/ML, Databases, or Security) and/or the industry (such as Retail, Healthcare, and so on) that this article would fit in.
```

File: Contribute/architecture-center/aac-contribute.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    1. A draft of your article (and/or your code) if you have it. The most efficient way to do this is to send us a Word doc. This could include a link to an accessible location (like a OneDrive), a blog post or GitHub MD file that the article is based on, and so on.
+    1. List the names of all the team members involved in creating this content. Include them in CC on your email.
+    
+1. We will follow up with you and ask any questions, to make sure this isn't repeated content, and to sync on the next steps.
+
+1. You (the contributor) submit the draft article. Because we don't have resources to create the draft, you must write the initial draft of the article.
+
+    - If this article is an architecture, you must provide a Visio or PowerPoint diagram of the architecture.
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:61]:
```suggestion
> [!NOTE]
> If this article is an architecture, you must provide a Visio or PowerPoint diagram of the architecture.
```

File: Contribute/architecture-center/aac-contribute.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+1. We will review the draft with you, to ensure technical accuracy and technical completeness (it tells the full story).
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:63]:
```suggestion
1. We will review the draft with you to ensure technical accuracy and technical completeness (it tells the full story).
```

File: Contribute/architecture-center/aac-contribute.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+1. We will edit the article (technical edit and copy edit), edit and refine the diagram, and publish the final article.
+
+## Edit an existing article
+
+You can edit an existing article on Azure Architecture Center, if you see a typo, want to add a link, or want to add more details.
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:69]:
```suggestion
You can edit an existing article on Azure Architecture Center if you see a typo, want to add a link, or want to add more details.
```

File: Contribute/architecture-center/aac-contribute.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+1. You (the contributor) must submit a proposed article. Send an email to [AACProposals@microsoft.com](mailto:AACProposals@microsoft.com). Include the following information:
+    1. Your name.
+    1. The name of your proposed article.
+    1. What type of content this should be. Options include an Architecture Guide, a Solution Idea, an Example Workload, or a Reference Architecture.
+    1. A brief description of your article.
+    1. The category (such as AI/ML, Databases, or Security) and/or the industry (such as Retail, Healthcare, and so on) that this article would fit in (
+    1. A draft of your article (and/or your code) if you have it. The most efficient way to do this is to send us a Word doc. This could include a link to an accessible location (like a OneDrive), a blog post or GitHub MD file that the article is based on, and so on.
+    1. List the names of all the team members involved in creating this content. Include them in CC on your email.
+    
+1. We will follow up with you and ask any questions, to make sure this isn't repeated content, and to sync on the next steps.
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:57]:
```suggestion
1. We will follow up with you and ask any questions to make sure this isn't repeated content and to sync on the next steps.
```

File: Contribute/architecture-center/aac-contribute.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+- Be technically accurate.
+- Provide helpful or valuable information in the context of the article.
+
+All edits *must not have* the following:
+
+- Link to advertise another company's product, unless the company's components are part of the architecture.
+
+### Process
+
+**Setup**. Steps to follow to set up your GitHub account and browser for editing:
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:84]:
```suggestion
**Setup**. Steps you must follow to set up your GitHub account and browser for editing:
```

File: Contribute/architecture-center/aac-contribute.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+1. Set up your GitHub account. See [GitHub account setup](/contribute/get-started-setup-github).
+
+1. To improve your edit experience, install SpineEdit from the [Chrome web store](https://chrome.google.com/webstore/detail/spineedit/llhlgkbkfdfcbjbfnnakfpgmemopbbnf). The extension works in all Chromium-based browsers, including Microsoft Edge.
+
+Learn more about the benefits of [SpineEdit](https://github.com/craigshoemaker/spineedit#introduction).
```
Comment (jehchow) [inline Contribute/architecture-center/aac-contribute.md:90]:
```suggestion
To learn more about the benefits of SpineEdit, see [SpineEdit introduction](https://github.com/craigshoemaker/spineedit#introduction). 
```

File: Contribute/architecture-center/guide-templates.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+- Links to related Azure Architecture Center articles.
+- Links should be relative, for example (../../solution-ideas/articles/article-name.yml).
+
+```
+
+## Browse format for Architecture Guides
+
+Browse format articles must have at least four files:
+
+- A YML file that has the metadata and article H1 title
```
Comment (jehchow) [inline Contribute/architecture-center/guide-templates.md:84]:
```suggestion
- A YML file that has the metadata and article H1 title.
```

File: Contribute/architecture-center/guide-templates.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+- The *-content.md* file that has the article content
```
Comment (jehchow) [inline Contribute/architecture-center/guide-templates.md:85]:
```suggestion
- The *-content.md* file that has the article content.
```

File: Contribute/architecture-center/guide-templates.md [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+- One or more architectural diagrams, in PNG or SVG format, in the article
```
Comment (jehchow) [inline Contribute/architecture-center/guide-templates.md:86]:
```suggestion
- One or more architectural diagrams, in PNG or SVG format, in the article.
```

File: Contribute/architecture-center/guide-templates.md [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+- A browse thumbnail graphic uploaded to the */docs/browser/thumbs/* directory of the AAC repository
```
Comment (jehchow) [inline Contribute/architecture-center/guide-templates.md:87]:
```suggestion
- A browse thumbnail graphic uploaded to the */docs/browser/thumbs/* directory of the AAC repository.
```

**Review by jehchow — COMMENTED** [review]
@EdPrice-MSFT A few suggestions and edits before we merge. Thank you!

File: Contribute/architecture-center/aac-contribute.md (hunk 4 above)
Comment (EdPrice-MSFT) [reply Contribute/architecture-center/aac-contribute.md]: Great. I think I got these. Please review. Thanks!

**Comment by EdPrice-MSFT** [comment]
Thanks for the review! Sorry for the late response. We had a reorg and then vacation time. 

#Sign-Off

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

Please review, @Jim-Parker, @carlyrevier, and @jehchow.

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

We're trying to get this published.

@Kiranchandratrey

**Review by jehchow — APPROVED** [review]

**Pull request merged**